src/main/java/me/contaria/seedqueue/SeedQueue.java

```diff
@@ -190,14 +235,20 @@ private static long getGeneratingCount() {
      * @see SeedQueueConfig#shouldUseWall
      */
     private static long getGeneratingCount(boolean treatScheduledAsPaused) {
-        long count = SEED_QUEUE.stream().filter(entry -> !((treatScheduledAsPaused && entry.isScheduledToPause()) || entry.isPaused())).count();
+        long count = 0;
+        for (SeedQueueEntry entry: SEED_QUEUE) {
+            boolean isScheduled = treatScheduledAsPaused && entry.isScheduledToPause();
```

Comment:
`isScheduled` is not really accurate here since that would fit the result of `SeedQueueEntry#isScheduledToPause`. I don't think there is an issue with just keeping the if-statement a oneliner like previously in the stream